Maestro/subscriptions.json

```diff
@@ -317,6 +378,24 @@
         }
       }
     },
+    // Update dependencies in core-setup release/2.0.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/2.0.0/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/release/2.0.0/Latest.txt",
+        // Roslyn 1.0.0
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/roslyn/netcore1.0/Latest_Packages.txt"
```

Comment:
Please also remove it from `master` above.

---
In reply to: [112553301](https://github.com/dotnet/versions/pull/145#discussion_r112553301) [](ancestors = 112553301)